code/controllers/subsystem/economy.dm

```diff
@@ -32,10 +37,12 @@ SUBSYSTEM_DEF(economy)
 	dep_cards = SSeconomy.dep_cards
 
 /datum/controller/subsystem/economy/fire(resumed = 0)
+	var/delta_time = wait / (5 MINUTES)
 	for(var/A in bank_accounts)
 		var/datum/bank_account/B = A
 		B.payday(1)
-
+	var/effective_mailcount = living_player_count()
+	mail_waiting += clamp(effective_mailcount, 1, MAX_MAIL_PER_MINUTE * delta_time)
```

Comment:
Is this actually mail per minute, or mail per 5 minutse? Wouldn't mail per minute be (wait / 1 MINUTES) rather than (wait / 5 MINUTES)?

Reply:
honest truth, I have genuineley no clue how this operate, this was a leftover from the acronomy I adapted when I first did the PR, now that I'm looking at it again it feels it should be something else, since it's the number of mail items generated